nixos/modules/services/x11/desktop-managers/gnome.md

```diff
@@ -116,6 +116,37 @@ Unfortunately, we lack a way for these to be managed in a completely declarative
 So you have to enable them manually with an Extensions application.
 It is possible to use a [GSettings override](#sec-gnome-gsettings-overrides) for this on `org.gnome.shell.enabled-extensions`, but that will only influence the default value.
 
+## Nautilus Extensions {#sec-gnome-nautilus-extensions}
+
+Nautilus extensions can add additional functionality to Nautilus, usually via the right-click menu. Some of these extensions are dynamically loaded from `$NAUTILUS_4_EXTENSIONS_DIR` environment variable, which is automatically set if [](#opt-services.gnome.core-utilities.enable) is true (enabled by default if gnome is enabled).
+
+Many Python based extensions can be found on the wild, and some of which are even packaged in Nixpkgs, (for example: `nautilus-open-any-terminal`). To load these extensions, you'll have to add the package that loads Python extensions for Nautilus:
+
+```nix
+environment.systemPackages = [
+  gnome.nautilus-python
+];
+```
+
+Some of these extensions, also include `gsettings` schemas, and adding them to `systemPackages` won't make them visible in `dconf-editor` or when you run `gsettings set`. An example of such an extension is yet again `nautilus-open-any-terminal`. Hence for this example, you have to add it in two places:
+
+```nix
+environment.systemPackages = with pkgs; [
+  gnome.nautilus-python
+  nautilus-open-any-terminal
+];
+services.xserver.desktopManager.gnome.extraGSettingsOverridePackages = with pkgs; [
+  nautilus-open-any-terminal
+];
+```
```

Comment:
This should probably go to a separate section, as it is not specific to GNOME extensions. Though we will hopefully remove the option soon in favour of dconf module so perhaps it is not needed.